package.json

```diff
@@ -246,7 +254,7 @@
     "sade": "^1.8.1",
     "sinon": "^9.2.3",
     "sinon-chai": "^3.7.0",
-    "typescript": "^4.9.5",
+    "typescript": "^5.6.3",
```

Comment:
One last minor thing: do we want to jump the this all the way to 5.6? One somewhat useful feature of using a less-than-current version of TS is that it's a warning light when we/a contributor goes to use a newer feature that'd require end users upgrade their TS version just to consume our types.

Right now we have an actual minimum (on the v5.1+ file) of 5.1 but that can easily & accidentally change without us realizing it if the TS version is set to 5.6 here.

Just a thought though, up to you if we want to set this as the new minimum.